client_file_status_extended.go

```diff
@@ -0,0 +1,29 @@
+package smartling
+
+import "fmt"
+
+const (
+	endpointFileStatusExtended = "/files-api/v2/projects/%s/locales/%s/file/status"
+)
+
+// GetFileStatus returns extended file status for a file for a particular locale
+func (client *Client) GetFileStatusExtended(
```

Comment:
Network overhead eats significant time. I would recommend to change your application code to do a single bulk status request (all locales at once) and then work with particular locales.

If you still need this method then i would like to ask to rename it `GetFileLocaleStatus`, and `FileStatusExtended` -> `FileLocaleStatus`